scripts/pre_commit_linter.py

```diff
@@ -129,7 +154,7 @@ def _lint_js_files(node_path, jscs_path, config_jscsrc, input_dir):
     if not files_to_lint:
         print 'There are no JavaScript files to lint. Exiting.'
         print '----------------------------------------'
-        sys.exit(0)
+        return
```

Comment:
I have a feeling sys.exit(0)/sys.exit() is the safe to end the process with no error @seanlip  what do you think?